src/test/kotlin/no/nav/familie/ef/sak/service/OppgaveServiceTest.kt

```diff
@@ -0,0 +1,181 @@
+package no.nav.familie.ef.sak.service
+
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
+import no.nav.familie.ef.sak.integration.OppgaveClient
+import no.nav.familie.ef.sak.integration.dto.familie.Arbeidsfordelingsenhet
+import no.nav.familie.ef.sak.repository.BehandlingRepository
+import no.nav.familie.ef.sak.repository.FagsakRepository
+import no.nav.familie.ef.sak.repository.OppgaveRepository
+import no.nav.familie.ef.sak.repository.domain.*
+import no.nav.familie.ef.sak.repository.domain.Oppgave
+import no.nav.familie.kontrakter.felles.oppgave.*
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.data.repository.findByIdOrNull
+import java.net.URI
+import java.time.LocalDate
+import java.util.*
+
+internal class OppgaveServiceTest {
+
+
+    private var oppgaveClient = mockk<OppgaveClient>()
+
+    @MockK
```

Comment:
@MockK er en annotasjon for å injecte mocks. Det gir ingen mening å bruke den når du oppretter mocks selv.

Reply:
True that!